README.md

```diff
@@ -12,7 +12,7 @@ Greenwood is a modern and performant static site generator supporting Web Compon
 > Greenwood is currently a work in progress and we would love any help getting it to a 1.0 release.  Check out our [roadmap](https://github.com/ProjectEvergreen/greenwood/projects) to see what we're working on next and feel free to reach out through our [issue tracker](https://github.com/ProjectEvergreen/greenwood/issues) if you have any issues or would like to request a feature!
 
 ## Getting Started
-Our website has a complete Getting Started section that will walk you through creating a Greenwood project from scratch.
+Our website has a complete [Getting Started](http://www.greenwoodjs.io/getting-started) section that will walk you through creating a Greenwood project from scratch.
```

Comment:
Should probably at least document the CLI commands in the README as well, after the Installation step.  (Usage?).  

Maybe mention `npx` somewhere?